examples/relm4-components/file_dialogs.rs

```diff
@@ -91,15 +91,15 @@ impl SimpleComponent for App {
         sender: &ComponentSender<Self>,
     ) -> ComponentParts<Self> {
         let open_dialog = OpenDialog::builder()
-            .transient_for(root)
+            .update_root(|dialog| dialog.set_transient_for(Some(root)))
```

Comment:
This change is rather confusing regarding the API. I think adding a method like `transient_for_native` could make sense. What do you think?

Reply:
Sure, I'll add it. Too bad we can't implement both native and non-native dialogs in one method.